packages/vetrina/src/Vetrina/Products/Premium.purs

```diff
@@ -13,13 +13,13 @@ hblPremium =
   , description:
       DOM.div
         { id: "tb-paywall--product-text-hbl"
-        , className: "vetrina--product-text text-center text-base leading-tight px-5 pb-5"
```

Comment:
Should we remove the `vetrina--something` classes also from the DOM since most (all?) of them have been axed from Vetrina.less. I'm thining it can get confusing later on when someone, like "Why is this class here, where are its rules defined?"

Reply:
Yes, we talked about that but then if we only have the Tailwind classes in the DOM, then it makes debugging really hard. Or at least I haven't found an easy way for that..